cmake-modules/AzureVcpkg.cmake

```diff
@@ -54,13 +54,13 @@ macro(az_vcpkg_portfile_prep targetName fileName contentToRemove)
 endmacro()
 
 macro(az_vcpkg_export targetName macroNamePart dllImportExportHeaderPath)
-  # CONTROL file has an extra '#' in the comment section, because VcPkg can't handle an empty line at that position,
+  # The vcpkg.json file has an extra '#' in the comment section, because VcPkg can't handle an empty line at that position,
```

Comment:
@antkmsft I don't think this holds true anymore, since we are not using a `CONTROL` file.